server/src-lib/Hasura/Prelude.hs

```diff
@@ -116,13 +119,20 @@ afold = getAlt . foldMap pure
 bsToTxt :: B.ByteString -> Text
 bsToTxt = TE.decodeUtf8With TE.lenientDecode
 
+lbsToTxt :: BL.ByteString -> Text
+lbsToTxt = bsToTxt . BL.toStrict
+
```

Comment:
Likewise: this function is not used in this PR AFAICT?

Reply:
I have removed this now.